### PR TITLE
qemu: Fix travis build failure for Power

### PR DESCRIPTION
**Pull request by bpradipt**
Fixes: #2751

Signed-off-by: bpradipt@in.ibm.com

### CODE REVIEW

**Comment by bpradipt** [comment]
Re-pushed but don't see any changes w.r.to alignment. The file view shows correct [alignment](https://github.com/kata-containers/runtime/blob/a5272f7ae344f196edfdf668c233dd0726f8b5e4/virtcontainers/qemu_ppc64le.go)

**Comment by bpradipt** [comment]
> @bpradipt, "time" and "fmt" do not seem to be aligned.
> 
> I wonder when this regression was introduced.
I think this is from the following commit - 
https://github.com/kata-containers/runtime/commit/66b54f88c5f4023bbd69c02d0b3274e8567e8b35

File: virtcontainers/qemu_ppc64le.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ package virtcontainers
 
 import (
 	"time"
+        "fmt"
```
Comment (devimc) [inline virtcontainers/qemu_ppc64le.go:10]: did you run "go fmt" on this file?

Comment (bpradipt) [reply virtcontainers/qemu_ppc64le.go:10]: Thanks @devimc. Running `go gmt` reordered the changes and it looks proper now

**Comment by bpradipt** [comment]
@devimc @fidencio please have a look now..

**Comment by fidencio** [comment]
> /test-arm

Oops, sorry, wrong PR. :-/

**Comment by devimc** [comment]
waiting for travis...., I'm going to close and reopen this (I hope I can reopen it),

**Pull request merged**